hibernate-envers/src/main/java/org/hibernate/envers/internal/entities/mapper/relation/component/MiddleEmbeddableComponentMapper.java

```diff
@@ -52,6 +52,10 @@ public Object mapToObjectFromFullMap(
 			);
 			return componentInstance;
 		}
```

Comment:
An alternative is to remove the try-catch. 

I chose to keep the try-catch and just throw the ```AuditException``` because I was concerned that ```delegate.mapToEntityFromMap``` could throw an ```Exception``` that should be wrapped by the ```AuditException```.